src/test/java/com/microsoft/sqlserver/testframework/Constants.java

```diff
@@ -29,6 +29,7 @@ private Constants() {}
      * reqExternalSetup  - For tests requiring external setup
      * clientCertAuth  - - For tests requiring client certificate authentication setup
      * Fedauth - - - - - - For Fedauth tests
+     * JSONTest -  - - - - For tests requiring JSON setup

```

Comment:
why do we need to add a separate tag for this? this should always run along with the other datatype tests

Reply:
@lilgreenbird We would need to exclude these tests against test endpoints that don't support JSON. (I'm kind of surprised we don't have one for Data Classification, for example. But it looks like we've used SQL Server version for that, which isn't quite proper.)

Reply:
we need a new tag for new SQL Server version that supports JSON. (SQL Server 2023 or ?)

Reply:
yeah, why not? data classification was in 2019 only